aten/src/ATen/native/mps/operations/Indexing.mm

```diff
@@ -20,14 +19,177 @@
 #include <c10/util/irange.h>
 #include <c10/core/QScheme.h>
 #include <c10/util/SmallVector.h>
+#include <ATen/native/IndexKernel.h>
 
 #ifdef __OBJC__
 #include <MetalPerformanceShaders/MetalPerformanceShaders.h>
 #endif
 
+extern const char * index_select_kernel_mps;
```

Comment:
Another Nit, can we avoid these and have an inline function which returns you the library 👍 
```
inline getIndexingLibrary() {
   id<MTLLibrary> indexingLibrary = [device newLibraryWithSource: [NSString stringWithCString:               index_select_kernel_mps encoding:NSASCIIStringEncoding];
   return indexingLibrary;
}
```
something like that from `IndexKernels.mm`.